test/tests/meshgenerators/file_mesh_generator/tests

```diff
@@ -21,4 +21,27 @@
     design = 'meshgenerators/FileMeshGenerator.md'
     issues = '#14916'
   [../]
+
+  [./pre_checkpoint_load_test]
+    type = 'RunApp'
+    input = '2d_diffusion_test.i'
+    expect_out = 'Solve Converged'
+    cli_args = 'Outputs/checkpoint=true'
+    recover = false
+    requirement = 'The system shall have the ability to output checkpoint files along with the mesh meta data.'
+    design = 'meshgenerators/FileMeshGenerator.md'
+    issues = '#16192'
+  []
+
+  [./checkpoint_load_test]
+    type = 'Exodiff'
+    input = '2d_diffusion_test.i'
+    exodiff = '2d_diffusion_test_out.e'
+    cli_args = 'Mesh/inactive=square Mesh/fmg/type=FileMeshGenerator Mesh/fmg/file=2d_diffusion_test_out_cp/0001_mesh.cpr'
+    recover = false
+    prereq = pre_checkpoint_load_test
+    requirement = 'The system shall have the ability to load the mesh from checkpoint files.'
```

Comment:
Should this requirement include meta data in it?

Reply:
No because it does. We have no way currently to load the meta data in a separate mesh generator without know what data types are stored in the binary file.